website/docusaurus.config.js

```diff
@@ -45,6 +45,7 @@ const config = {
         docs: {
           path: '../docs',
           sidebarPath: require.resolve('./sidebars.js'),
+          exclude: ['https://wix.github.io/Detox/docs/demo'],
```

Comment:
So, this syntax is outdated, reportedly. Let's remove this, then, right?

Reply:
Hi - no, this is the new way to do it. the outdated/unsupported way is to use `ignorePatterns`.
We should leave this as-is with `exclude`

Reply:
I think this is incorrect. 

```suggestion
```

According [to the website](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#include), `include` means:

> Array of glob patterns matching Markdown files to be built, relative to the content path.

While `exclude` is the opposite. Yet **we want this file to be BUILT**. All we didn't want was just to get it indexed – that's why we excluded it from robots in head->meta element.